fortytools.yaml

```diff
@@ -1494,7 +1493,7 @@ paths:
       tags:
       - Oauth
       description: |
-        Authenticate client ID and client secret to get access token
+        Get access token
```

Comment:
It's fine to keep the long line as it was it you want, redocly will display the whole line underneath the title anyway
<img width="718" alt="Screenshot 2024-08-09 at 09 23 02" src="https://github.com/user-attachments/assets/2ec6768f-00d6-477e-b0aa-f74d4f685731">